### DISCUSSION TITLE
Error on Directx 11 on Windows 7 Ultimate x64

### DISCUSSION THREAD

**Discussion by Uranya** [WindowTop/WindowTop-App#74] [Q&A]
## **Dear developer,**
thank you for your great work, your program have saved my eyes with the dark mode!

I have now updated from the 4.1.3 to the new 5.1.9 version

But now, every time I turn on a function I get this error:

(Window name is Renderer.exe)
Impossible to find entrance point
CreateDirect3D11DeviceFromDXGIDevice in the procedure of the dynamic link d3d11.dll

This happens even in a setup than in a portable application.
I have DirectX 11 correctly installed and working.
Can you please help me to solve this issue?

Thank you!

**Comment by gileli121** [comment]
Hi, I'm surprised the software even works on Windows 7.
I never planed support for Windows 7 in v5.x.x because I think that most people already moved on and Windows 7 is discontinued by Microsoft.
I am sorry but I am not going to fix any issues with Windows 7.

You can use the old versions instead
